runtime/compiler/control/OptionsPostRestore.cpp

```diff
@@ -280,13 +280,16 @@ void
 J9::OptionsPostRestore::processJitServerOptions()
    {
 #if defined(J9VM_OPT_JITSERVER)
-   if (_argIndexUseJITServer >= _argIndexDisableUseJITServer)
+   bool jitserverEnabled
+      = (_argIndexUseJITServer > _argIndexDisableUseJITServer)
```

Comment:
At the moment, because we didn't enable jitserver at bootstrap, we don't do any of the set up to actually have the current JVM be a client. So even when you try to enable it post-restore, it doesn't look like anything happens. However, for the sake of conceptual integrity, I suppose I should explicitly cover this case. if we really want to enable jitserver in this scenario post restore, I can look into that in another PR.

Reply:
Addressed in latest force push.